src/JustSaying/MessagingBusBuilder.cs

```diff
@@ -229,6 +229,32 @@ public IMessagePublisher BuildPublisher()
         return bus;
     }
 
+    /// <summary>
+    /// Creates a new instance of <see cref="IMessageBatchPublisher"/>.
+    /// </summary>
+    /// <returns>
+    /// The created instance of <see cref="IMessagePublisher"/>
+    /// </returns>
+    public IMessageBatchPublisher BuildBatchPublisher()
+    {
+        IMessagingConfig config = MessagingConfig.Build();
+
+        config.Validate();
+
+        var publishBatchConfiguration = MessagingConfig.BuildPublishBatchConfiguration();
+        ILoggerFactory loggerFactory = ServiceResolver.ResolveService<ILoggerFactory>();
+
+        JustSayingBus bus = CreateBus(config, loggerFactory, publishBatchConfiguration);
+        IAwsClientFactoryProxy proxy = CreateFactoryProxy();
+
+        if (PublicationsBuilder != null)
+        {
+            PublicationsBuilder.Configure(bus, proxy, loggerFactory);
+        }
```

Comment:
Can this be simplified to
```suggestion
        PublicationsBuilder?.Configure(bus, proxy, loggerFactory);
```
?